libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-roles.component.html

```diff
@@ -0,0 +1,64 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *rxLet="loading as isLoading">
+  <watt-card
+    variant="solid"
+    *transloco="let t; read: 'admin.userManagement.permissionDetail.tabs.userRoles'"
+  >
+    <div *ngIf="isLoading" class="spinner">
+      <watt-spinner></watt-spinner>
+    </div>
+
+    <ng-container *ngIf="!isLoading">
+      <watt-card-title *rxLet="userRoles?.length as userRoleCount">
+        <h4 *ngIf="userRoleCount === 1 && !error">
+          {{t("rolesSingular", {userRoleCount})}}
+        </h4>
+        <h4 *ngIf="userRoleCount !== 1 && !error">
+          {{t("rolesPlural", {userRoleCount})}}
+        </h4>
+      </watt-card-title>
+
+      <watt-table
+        [columns]="columns"
+        [dataSource]="dataSource"
+        sortBy="name"
+        sortDirection="desc"
+        [sortClear]="false"
+        [resolveHeader]="translateHeader"
```

Comment:
suggestion: the `resolveHeader` property is marked as "deprecated". It's recommended to use the `header` Input on the `wattTableCell` directive. See "dh-users-tab-table.component.html" for an example.

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/pull/1583/commits/36fdc0c95e1fffdc0e3b9bd2a5dbbbcf51cb043b